peewee_async/pool.py

```diff
@@ -2,7 +2,10 @@
 import asyncio
 from typing import Any, Optional, cast
 
-from .utils import aiopg, aiomysql, PoolProtocol, ConnectionProtocol
+import psycopg_pool
+from psycopg import AsyncClientCursor
```

Comment:
now psycopg_pool and psycopg are requirements.

You must import it from `.utils` where you can place safe import with try/except